youtube_dl/extractor/azmedientv.py

```diff
@@ -0,0 +1,110 @@
+        'info_dict': {
+            'id': '1_ii9ckh5g',
+            'ext': 'mov',
+            'title': 'Zu wenig Pflegerinnen und Pfleger',
+            'description': 'md5:bf7e2abd614bd83958304ae033d5c30d',
+            'uploader_id': "TeleB?rn",
+            'upload_date': '20170117',
+            'timestamp': 1484677297,
+        },
+        'params': {
+            'skip_download': True,
+        }
+    }, {
+        # URL with 'episode' and fragment:
+        'url': 'http://www.telem1.ch/47-show-sonntalk/13986-episode-soldaten-fuer-grenzschutz-energiestrategie-obama-bilanz#soldaten-fuer-grenzschutz-energiestrategie-obama-bilanz',
+        'info_dict': {
+            'id': '1_s4r1b250',
+            'ext': 'mp4',
+            'title': 'Soldaten f\u00fcr Grenzschutz \u2014 Energiestrategie \u2014 Obama-Bilanz',
+            'description': 'md5:f6327ccd57d4477525d67d320b5ae7c1',
+            'uploader_id': "TVOnline",
+            'upload_date': '20170101',
+            'timestamp': 1483296106,
+        },
+        'params': {
+            'skip_download': True,
+        }
+    }, {
+        # URL with 'show' and fragment:
+        'url': 'http://www.telezueri.ch/66-show-sonntalk#burka-plakate-trump-putin-china-besuch',
+        'only_matching': True
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        kaltura_partner_id = self._html_search_regex(
+            r'<script[^>]+src=["\']https?://www\.kaltura\.com/.*/partner_id/([0-9]+)',
+            webpage, 'Kaltura partner ID')
+        kaltura_entry_id = self._html_search_regex(
+            r'<a[^>]+data-id=["\'](.*?)["\'][^>]+data-slug=["\']%s' % video_id,
+            webpage, 'Kaltura entry ID')
+
+        return self.url_result(
+            'kaltura:%s:%s' % (kaltura_partner_id, kaltura_entry_id),
+            ie=KalturaIE.ie_key())
+
+
+class AZMedienTVShowIE(InfoExtractor):
+    IE_DESC = 'telezueri.ch, telebaern.tv and telem1.ch shows'
+    _VALID_URL = r'http://(?:www\.)?(?:telezueri\.ch|telebaern\.tv|telem1\.ch)/(?P<id>[0-9]+-show-[^/#]+(?:/[0-9]+-episode-[^/#]+)?)$'
+
+    _TESTS = [{
+        # URL with 'episode':
+        'url': 'http://www.telebaern.tv/118-show-news/13735-episode-donnerstag-15-dezember-2016',
+        'info_dict': {
+            'id': '118-show-news/13735-episode-donnerstag-15-dezember-2016',
+            'title': 'News',
+        },
+        'playlist_count': 9,
+    }, {
+        # URL with 'show' only:
+        'url': 'http://www.telezueri.ch/86-show-talktaeglich',
+        'only_matching': True
+    }]
+
+    def _real_extract(self, url):
+        show_id = self._match_id(url)
+        webpage = self._download_webpage(url, show_id)
+
+        title = str_or_none(
+            get_element_by_class('title-block-cell', webpage)).strip()
```

Comment:
`get_element_by_class` is already a string or `None`. `strip` on `None` will break.